### DISCUSSION TITLE
What are the assigner-leader-<id> topics?

### DISCUSSION THREAD

**Discussion by Cave-Johnson** [strimzi/strimzi-kafka-operator#6678] [Q&A]
More a general question, i am running the latest version of Strimzi. When I run my producers and consumers (faust-streaming python apps) I get these topics generated called `<topic-name>-assigner-leader-<some hash id>`.

Are these strimzi related and if so what are they used for? Apologies if these are faust related! Still on the learning journey!

**Comment by scholzj** [comment]
I never saw these. So I guess they are maybe related to Faust-streaming?

**Reply by Cave-Johnson** [reply]
Thanks for that! I'll have a dig in to the faust side
